deployment/ansible/group_vars/all

```diff
@@ -30,6 +30,7 @@ postgis_package_version: "2.3.1+dfsg-1.pgdg14.04+1"
 # azavea.docker
 # 1.11.* to match AWS ECS
 docker_version: 1.11.*
+docker_options: "--storage-opt dm.basesize=20G"
```

Comment:
Exactly. In a separate branch this AM I was able to setup an umanaged compute environment and attach ECS autoscaling/instance launch configs that automatically map an EBS volume on boot via cloud init. So I'll play with this more today over there on that branch rather than here.

I can remove the storageopts setting as that's probably not necessary with the other coming changes.